shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/statement/ShardingSphereStatement.java

```diff
@@ -385,7 +390,11 @@ protected Boolean getSaneResult(final SQLStatement sqlStatement) {
         };
         return driverJDBCExecutor.execute(executionGroups, sqlStatement, routeUnits, jdbcExecutorCallback);
     }
-    
+
+    private void isToCalcite(final boolean isToCalcite) {
```

Comment:
Hi, we can not add `set` or `get` functions for tests. I guess you want to make this property `true`. That way, please refer to the following steps,
- Add another single table to `config-calcite.yaml`
- Modify your test SQL as `select xx,xx from singleTable1, singleTable2 where singleTable1.xx=singleTable2 and xx`
- If necessary, you can add SQLS to prepare the Calcite UT in `jdbc_data.sql` and `jdbc_init.sql`. It is straightforward to create tables to do calcite tests, isn't it?

Reply:
> Hi, we can not add `set` or `get` functions for tests. I guess you want to make this property `true`. That way, please refer to the following steps,
> 
> * Add another single table to `config-calcite.yaml`
> * Modify your test SQL as `select xx,xx from singleTable1, singleTable2 where singleTable1.xx=singleTable2 and xx`
> * If necessary, you can add SQLS to prepare the Calcite UT in `jdbc_data.sql` and `jdbc_init.sql`. It is straightforward to create tables to do calcite tests, isn't it?

Thanks for your encouragement, you are so kind.  I have a  questions, how to go the branch as follows when test？It seems nowhere can set the boolean value `toCalcite` in `RouteContext`
```
 if (executionContext.getRouteContext().isToCalcite()) {
            return executeQueryByCalcite();
        }
```

Reply:
Hi, please look at Line 57 in `SingleTableRoutingEngine.java`

Reply:
That's why I suggest you add another single table to config-calcite.yaml and modify your test SQL.